src/enc.rs

```diff
@@ -188,14 +190,17 @@ pub trait Encoder {
     ) -> Result<Self::Ok, Self::Error>;
 
     /// Encode a `SEQUENCE` value.
-    fn encode_sequence<C, F>(
+    ///
+    /// Const `N` is the count of root components in sequence or set.
+    /// Generic `C` is the sequence value.
+    fn encode_sequence<const N: usize, C, F>(
```

Comment:
Couldn't this just be C::Constructed::FIELDS.len()

Reply:
Currently you can't, the problem is similar as in #318 

You will get a compiler error:
```sh
error[E0401]: can't use generic parameters from outer item
   --> src/oer/enc.rs:849:28
    |
840 |     fn encode_sequence<const N: usize, C, F>(
    |                                        - type parameter from outer item
...
849 |         const LEN: usize = C::FIELDS.len();
    |                            ^^^^^^^^^ use of generic parameter from outer item
    |
    = note: a `const` is a separate item from the item that contains it

error: generic parameters may not be used in const operations
   --> src/oer/enc.rs:850:39
    |
850 |         let mut encoder = Encoder::<{ C::FIELDS.len() }>::new(
    |                                       ^^^^^^^^^ cannot perform const operation using `C`
    |
    = note: type parameters may not be used in const expressions
```